packages/fast-navigation-generator-react/src/navigation/navigation.style.ts

```diff
@@ -0,0 +1,78 @@
+import { ComponentStyles, CSSRules } from "@microsoft/fast-jss-manager";
+import { applyTriggerStyle, insetStrongBoxShadow } from "./navigation.utilities.style";
+import {
+    accent,
+    background100,
+    background800,
+    foreground300,
+} from "./navigation.constants.style";
+
+export interface NavigationClassNameContract {
+    navigation?: string;
+    navigation_item?: string;
+    navigation_itemLink?: string;
+    navigation_itemLink__active?: string;
+    navigation_itemExpandListTrigger?: string;
+    navigation_itemExpandListTrigger__active?: string;
+    navigation_itemList?: string;
+}
+
+const styles: ComponentStyles<NavigationClassNameContract, {}> = {
+    navigation: {
+        fontSize: "11px",
+        background: background100,
+        height: "100%",
+    },
+    navigation_item: {
+        display: "flex",
+        flexFlow: "column",
+        textIndent: "1em",
+        '&[aria-expanded="true"] > $navigation_itemList': {
+            display: "block",
+        },
+        '&[aria-expanded="false"] > $navigation_itemExpandListTrigger::before, &[aria-expanded="true"] > $navigation_itemExpandListTrigger::before': {
+            content: "''",
+            display: "inline-block",
+            marginRight: "5px",
```

Comment:
Just verifying, no localization?

Reply:
Not at this time, the tooling will need localization all-up as it has not been adequately supported in any tooling package.